tests/config/write.c

```diff
@@ -670,7 +670,6 @@ void test_config_write__locking(void)
 	git_transaction_free(tx);
 
 	/* Now that we've unlocked it, we should see both updates */
-	cl_git_pass(git_config_open_ondisk(&cfg, filename));
```

Comment:
We do want to load the file independently to test that the changes are visible from the outside. I would add a `git_config_free(cfg)` after `git_transaction_free()` instead.